WMF Framework/Remote Notifications/Operations/RemoteNotificationsMarkAsReadOperation.swift

```diff
@@ -0,0 +1,21 @@
+class RemoteNotificationsMarkAsReadOperation: RemoteNotificationsOperation {
+    override func execute() {
+        DispatchQueue.main.async {
```

Comment:
What's the reason for the main queue async here and in RemoteNotificationsFetchOperation?

Reply:
don't I need to kick off operations on private queues from the main queue?

Reply:
Not necessarily. We're doing it elsewhere where it triggers creation of a background moc from the viewContext, which we want to limit to occurring on the main thread. In this case, since the moc exists already, it should be unnecessary.

Reply:
Ok, makes sense 👍